automation/run-tests.sh

```diff
@@ -191,8 +191,11 @@ function is_file_changed {
 
 function rebuild_container_images {
     if is_file_changed "$PROJECT_PATH/packaging"; then
-
-        ${PROJECT_PATH}/packaging/build-container.sh all
+        if [ $CONTAINER_IMAGE == $CENTOS_IMAGE_DEV ];then
```

Comment:
We need this change now as it seems that the F32 is somehow broken. Can you move this into its own PR?

Reply:
Sure.

Reply:
https://github.com/nmstate/nmstate/pull/816